spec/menu_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'menu'
+
+describe Menu do
+  it 'contains a list of dishes and prices' do
+    expect(subject.dishes).to eq ({:Noodles=>3.0, :Rice=>2.5})
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.